substrate/frame/revive/Cargo.toml

```diff
@@ -33,6 +33,7 @@ serde = { features = [
 	"alloc",
 	"derive",
 ], workspace = true, default-features = false }
+alloy-core = { workspace = true, features = ["sol-types"] }
```

Comment:
Move to top. Blocks should be ordered alphabetically. I am wondering why the fmt job doesn't catch this.